draft-ietf-oauth-selective-disclosure-jwt.md

```diff
@@ -1135,16 +1135,41 @@ To indicate that the content is an SD-JWT:
 * Applications that use this media type: TBD
 * Fragment identifier considerations: n/a
 * Additional information:
-   Magic number(s): n/a
-   File extension(s): n/a
-   Macintosh file type code(s): n/a
+   * Magic number(s): n/a
+   * File extension(s): n/a
+   * Macintosh file type code(s): n/a
 * Person & email address to contact for further information: Daniel Fett, mail@danielfett.de
 * Intended usage: COMMON
 * Restrictions on usage: none
 * Author: Daniel Fett, mail@danielfett.de
-* Change Controller: IESG
+* Change Controller: IETF
 * Provisional registration?  No
 
+<br/>
+To indicate that the content is a JWS JSON serialized SD-JWT:
```

Comment:
to be consistent and bring clarity, we could change line 1125 to `To indicate that the content is a JWS compact serialized SD-JWT:` but Brian convinced me that no need because we do not really use the term compact serialized in the text and it is kind of obvious without the text changes I considered.